### PR TITLE
Document CVE-2017-2672

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -69,6 +70,16 @@ All security advisories made for Foreman are listed below with their correspondi
...
 ### Disclosure details
 
+#### <a id="2017-2672"></a>CVE-2017-2672: audit log stores plain text compute resource image passwords
+
+Compute resource images added/registered in Foreman with username and password credentials will have the password stored in plain text in the audit log. A user with permission to view the audit log can view the password and may gain access to newly provisioned hosts.
+
+This issue was reported by Daniel Kimsey.
+
+* Affects Foreman 1.4 and higher
+* Fix in progress
```
Comment (ares) [inline security.md:80]: fix has been merged meanwhile https://github.com/theforeman/foreman/pull/4438, would you mind updating this?

Comment (domcleal) [reply security.md:80]: Normally this stays as "in progress" until it's released, or assigned a release.

Comment (ares) [reply security.md:80]: ok, I guess someone will assign a release number soon but if you want to send another PR later, I'm happy to merge it as is :-)

**Comment by ares** [comment]
Thanks @domcleal, merged.

**Pull request merged**